Library/Advantages/GURPS Lite Advantages.adq

```diff
@@ -0,0 +1,1582 @@
+<?xml version="1.0" encoding="US-ASCII" ?>
```

Comment:
This file should be renamed to take the "Advantages" part out of it, since that should be obvious given that it is within the Advantages folder.

Reply:
Done